src/styles/SearchForm.css

```diff
@@ -0,0 +1,24 @@
+@import url("https://fonts.googleapis.com/css2?family=Goldman&family=Press+Start+2P&display=swap");
+
+.SearchForm {
+  padding: 20px;
+  display: flex;
+  justify-content: space-between;
+}
+
+/* label {
+  font-size: 20px;
+  font-weight: bold;
+} */
```

Comment:
Should always remove commented out code like this, it just adds noise to the file